internal/common/rules/globalruleregistration_test.go

```diff
@@ -0,0 +1,16 @@
+package rules
+
+import "testing"
+
+// TestGlobalRuleRegistration checks whether required rules are registered
+func TestGlobalRuleRegistration(t *testing.T) {
+	rulesToFind := []string{
+		"Kinda Complicated Rule",
```

Comment:
fine for now, but would be nice if we have the file of constants for the names of rules as well, then we can have a nice self-contained unit test for `AvailableRules`

Reply:
Yup will be doing an overhaul of unit tests in later PR